src/components/AccessibilityGuide/RepetitiveActions.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import { Meta } from "@storybook/addon-docs";
+
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta title={getCategory("Repetitive Actions")} />
+
+# Repetitive Actions
+
+## Table of Contents
+
+- [General Information](#general-information)
+- [Unique Button Labels and Link Text](#unique-button-labels-and-link-text)
+- [NYPL Patterns](#nypl-patterns)
+- [Resources](#resources)
+
+## General Information
+
+To navigate a website, screen reader users often tab through buttons and links
+on a page or they simply pull up a list of all buttons and links on the page.
+When button labels and link text are not unique, it becomes difficult for these
+users to determine the purpose or destination of the button or link. For
+example, if you have multiple "read more" links on a page, these links are not
+descriptive of their destination without the user having to examine the
+surrounding content of each link.
+
+## Unique Button Labels and Link Text
+
+Button labels and link text should generally be unique on any given page,
+primarily to aid navigation for screen reader users.
+
+In cases where the visible text cannot be changed, additional context can be
+provided for screen reader users in several ways, including:
+
+- adding additional text within the button or link and hiding it off-screen
+  so sighted users will not perceive it, but it will be picked up by screen
+  reader software
+- using aria-label to override the visible text
+- using aria-labelledby to override the visible text by referencing both the
+  link itself and another element that will add more descriptive text
+
+For example, a link that is visibly titled "Read more" can:
+
+- include `<span class="sr-only">about jobs<?span>` after the visible text
+  within the link where `sr-only` is a custom class with styles to position the
+  `<span>` element off-screen
+- add the `aria-label` attribute with complete descriptive text
+- add the `aria-labelledby` attribute with references to multiple element IDs,
+  including the link itself and another more descriptive piece of text, such as
+  a nearby heading, resulting in a label such as "read more about jobs"
+
+### Examples:
+
+#### sr-only
```

Comment:
Should we include a snippet of CSS that shows what the `sr-only` class can be?

Reply:
Yes, that is a good idea.